iib/workers/tasks/build.py

```diff
@@ -133,7 +133,7 @@ def _create_and_push_manifest_list(request_id, arches):
     return output_pull_spec
 
 
-def _finish_request_post_build(
+def update_index_image_pull_spec(
```

Comment:
Could you please keep this as a private function?

Reply:
oh yes..

Reply:
done